click/core.py

```diff
@@ -1266,7 +1266,8 @@ class Parameter(object):
 
     def __init__(self, param_decls=None, type=None, required=False,
                  default=None, callback=None, nargs=None, metavar=None,
-                 expose_value=True, is_eager=False, envvar=None):
+                 expose_value=True, is_eager=False, envvar=None,
+                 autocompletion=None):
```

Comment:
If this is only available for arguments, shouldn't it be defined on that class instead?

Reply:
This is not documented in the `:param:` list above.